1-fundamentos/b03-intro-html-css/d02-primeiros-passos/exercicio1.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS Basico</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <!-- <style>
+        h1 {
+            font-size: 65px;
+            font-family: sans-serif;
+            color: orange;
+        }
+
+        ul {
+            background-color: blue;
+            text-align: left;
+            text-decoration: dotted;
+            line-height: 1.5em;
+        }
+
+        .list-p-style {
+            background-color: aquamarine;
+        }
+
+        p {
+            font-weight: 600;
+            font-size: 3em;
+            font-style: italic;
+        }
+    </style> -->
+</head>
+
+<body>
+    <h1>Exercícios 3.2</h1>
+    <p class="list-p-style">Qual é a sua cor favorita?</p>
+    <ul class="list-p-style">
+        <li>Amarelo</li>
+        <li>Vermelho</li>
+        <li>Marrom</li>
```

Comment:
```suggestion
        <li>Amarela</li>
        <li>Vermelha</li>
        <li>Marrom</li>
```
De acordo com o aurélio as cores são femininas.